commands/metrics.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'node:fs/promises'
+import { Tail } from 'tail'
+import { paths } from '../lib/paths.js'
+
+const metricsLogLineToJSON = (metrics) =>
+  JSON.stringify(
+    JSON.parse(metrics.split(/\[([^\]]+)\] /).pop()),
+    0,
+    2
+  )
```

Comment:
I'm currently not sure about this part. We're storing metrics as log lines prefixed by `[$date] `, and now parsing them in order to output them as JSON. If we say that `$ station metrics` will be the primary way to look at metrics, then I'd propose to use NLDJSON, just as pino uses. While not as convenient when inspecting the files manually, it is easier to integrate and work with. Wdyt @bajtos?

Reply:
I agree 👍  I've opened a new issue here for final discussion: https://github.com/filecoin-station/core/issues/37

I believe we can ship this PR without it for now